transports/quartz/src/main/java/org/mule/transport/quartz/QuartzMessageReceiver.java

```diff
@@ -117,7 +117,7 @@ protected void doStart() throws MuleException
 
             jobDetail.setJobDataMap(jobDataMap);
 
-            Trigger trigger;
+            AbstractTrigger trigger;
```

Comment:
again the same as in the dispatcher. COnsider having factory methods to create these things